microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -127,10 +137,24 @@ The Evergreen distribution mode ensures that your WebView2 app is taking advanta
 
 When you use the Evergreen distribution mode of the WebView2 Runtime, your WebView2 app assumes that clients have the latest Runtime.  Your app cannot require a particular version of the WebView2 Runtime for all apps on the client.  By the time a new WebView2 SDK package is released, a compatible version of the WebView2 Runtime has already been distributed to clients.  Therefore it's OK for your WebView2 app to use the APIs that are in the latest version of the WebView2 SDK.
 
-See [Understanding browser versions and WebView2](versioning.md).
+
+<!-- --------------- -->
+<!-- #### Evergreen WebView2 Runtime is preinstalled on Win11 devices and most Win10 devices -->
+
+The vast majority of Windows 10 devices have the WebView2 Runtime installed already.  See [Delivering Microsoft Edge WebView2 Runtime to managed Windows 10 devices](https://blogs.windows.com/msedgedev/2022/12/14/delivering-microsoft-edge-webview2-runtime-to-managed-windows-10-devices/).
```

Comment:
nit - I would probably this and next paragraph as a single paragraph instead of 2.

Reply:
concated